src/en/anchira/src/eu/kanade/tachiyomi/extension/en/anchira/Anchira.kt

```diff
@@ -109,6 +112,12 @@ class Anchira : HttpSource(), ConfigurableSource {
             fetchMangaDetails(manga).map {
                 MangasPage(listOf(it), false)
             }
+        } else if (query.startsWith(SLUG_BUNDLE_PREFIX)) {
```

Comment:
what about a filter entry (checkbox?) to toggle this.

Reply:
Haven't considered it. I avoid messing with filters because it breaks TachiSY's saved searches and adding `bundle:` is simple enough. I meant it as an advanced feature due to the extra calls, so the friction works in favor